SU-SWS/ace-sdssgryphon
# SDSS-434: mobile

# READY FOR REVIEW

# Summary
- Fixes to the main and dept. site headers in responsive views.
- The ticket has design images for comparison. 
- Design confirmed this one was a go.

# Review By (Date)
- 1/30

# Urgency
- Normal

# Steps to Test

1. Checkout the branch.
2. Make sure you have a main site and a dept site to test on.
3. Main site should not have blue in the header when it is sized MD down.
4. Dept sites have two different looks for MD and XS/SM. Those images are in the ticket. 
5. I've tested this on Chrome, FF and Safari on the Mac
6. I started the separation of header scss files.  That can continue at a later date. 

# Affected Projects or Products
- SDSS Subtheme

# Associated Issues and/or People
- [SDSS-434](https://stanfordits.atlassian.net/browse/SDSS-434)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)


[SDSS-434]: https://stanfordits.atlassian.net/browse/SDSS-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
